model/fv_arrays.F90

```diff
@@ -1251,6 +1254,10 @@ module fv_arrays_mod
     real, _ALLOCATABLE :: peln(:,:,:)   _NULL  !< ln(pe)
     real, _ALLOCATABLE :: pkz (:,:,:)   _NULL  !< finite-volume mean pk
 
+
+! For downscaling coupling variables from parent to nested grids
+    real, _ALLOCATABLE :: downcpl2d(:,:) _NULL !< temporary 2d arrary for downscaling coupling variables
```

Comment:
I am less familiar with the coupling code.  @BinLiu-NOAA can you comment here?

Reply:
@lharris4 This downcpl2d variable is currently used to downscale/pass the SST or sea surface roughness length (z0) variables from the FV3 parent grid to its nest grid. This is because currently the ocean/wave coupling only happens between the FV3atm parent domain and the ocean/wave domains. Thus, we need to pass the SST/z0 fields from the parent to the nest so that the nest domain also gets updated values from the ocean/wave model components. This is currently a temporary solution/workaround. The plan is to enable the coupling/exchange between the FV3atm nested domain and ocean/wave domains directly in future as a better solution.

Alternatively, if you do not want add this downcpl2d variable here. We can move it back inside the subroutine atmosphere_fill_nest_cpl as a temporary array. 

Please feel free to let us know your preference. Thanks!

Reply:
Hi, Bill, Bin. If it doesn't need to be preserved between calls, then it doesn't need to be in fv_array, and since it is physics-specific it probably doesn't belong here anyway.

Thanks,
Lucas

Reply:
@lharris4, Thanks for the comments/suggestions! I will move it inside the atmosphere_fill_nest_cpl subroutine as a temporary array at this time. And we can revisit this once the direct coupling between the FV3 nested grids and the wave/ocean domains are implemented/enabled. Thanks!

Reply:
Thanks. We will revisit in a future release.

Reply:
Thanks, @lharris4!